justfile

```diff
@@ -181,6 +181,24 @@ test-int: clean-test install-sqlx
         fi
     fi
 
+# Generate code coverage report
+coverage *ARGS="--open": clean start (cargo-install "cargo-llvm-cov")
+    #!/usr/bin/env bash
+    set -euo pipefail
+    if ! rustup component list | grep llvm-tools-preview > /dev/null; then \
```

Comment:
why do you need to manually install `llvm-tools-preview`?  Shouldn't the cargo-llvm-cov automatically do that?

Reply:
Yes and no.
They would, but since we execute this the first time in the env-variable step it is not printed to the screen that the user needs to press y..